### ISSUE TITLE
Array error on hook trigger

### ISSUE THREAD

**Issue by cchimi** [getkirby/kirby#451]
Hi, I installed the 2.3.0 update today and I'm now getting the following error on my panel.user.update hook:

" in_array () expects parameter 2 to be array, integer given in [my project location]/kirby/kirby.php on line 725"

which appears to refer to 

`in_array($key, static::$triggered[$hook])`

My code hasn't changed since the update. Is there anything new about how these should be set up? Let me know if code would be helpful here.

**Comment by lukasbestle** [comment]
This is a bug. I have created PR #452 to fix it. Thanks for reporting.

**Comment by jenstornell** [comment]
For me it appears when having too hooks that does the same thing:

```php
<?php
kirby()->hook('panel.page.update', function($page) {
});

kirby()->hook('panel.page.update', function($page) {
});
```

I tested the fix. I don't know yet if the hooks works but I can confirm that the panel will no longer hang.

**Comment by bembel-with-care** [comment]
Thank you for the feedback. It would be very helpful to know whether both hooks run. :)

**Comment by lukasbestle** [comment]
Thanks for the feedback. It would be very helpful to know whether both hooks run. :)

**Comment by jenstornell** [comment]
Alright. I tested it now and both my hooks run without a problem. I'm still using the old syntax, not kirby set.

I'm hoping for a minor bugfix release, Kirby 2.3.1.

**Comment by DavidUnzue** [comment]
I was having the same issue when using two hooks with the same ID. I can also confirm that the bugfix is working.

**Comment by mlbrgl** [comment]
@lukasbestle Same scenario, calling the same hook in different plugins failed. Both are now fired as expected. Thanks!

**Comment by lukasbestle** [comment]
The fix for this will be included in Kirby 2.3.1.
